src/main/java/org/fcrepo/auth/webac/WebACAuthorization.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.auth.webac;
+
+import java.net.URI;
+import java.util.Set;
+
+import org.fcrepo.kernel.api.models.FedoraResource;
+
+import com.hp.hpl.jena.vocabulary.RDF;
+
+/**
+ * Interface for WebAC ACL objects.
+ * @author whikloj
+ * @since 2015-08-25
+ */
+public interface WebACAuthorization {
+
+    /**
+     * Get the set of acl:agents, empty set if none.
+     *
+     * @return set of acl:agents
+     */
+    Set<String> getAgents();
+
+    /**
+     * Get the set of acl:agentClass URIs, empty set if none.
+     *
+     * @return set of acl:agentClass URIs
+     */
+    Set<URI> getAgentClasses();
+
+    /**
+     * Get the set of acl:modes, empty set if none.
+     *
+     * @return set of acl:modes
+     */
+    Set<URI> getModes();
+
+    /**
+     * Get the set of FedoraResources directly linked from this ACL, empty set if none.
+     *
+     * @return set of FedoraResource
+     */
+    Set<FedoraResource> getObjects();
+
+    /**
+     * Get the set of RDF describing the rdf:types for this ACL, empty set if none.
+     *
+     * @return set of RDF objects
+     */
+    Set<RDF> getObjectClasses();
```

Comment:
@whikloj The com.hp.hpl.jena.vocabulary.RDF class is (AFAICT) a static constants class that returns Jena Property or Resource objects for items in the RDF vocabulary (e.g., rdf:value, rdf:Seq). We could certainly use `RDF.type` from that class in a implementation to locate the type triples, but the returned classes would either be `URI` or ` com.hp.hpl.jena.rdf.model.Resource` objects. My vote is for `URI`, to keep from tying us at this level to Jena.